src/main/java/com/mycompany/cajeroautomatico/CajeroAutomatico.java

```diff
@@ -1,187 +0,0 @@
-
-package com.mycompany.cajeroautomatico;
-
-import java.io.IOException;
-
-public class CajeroAutomatico {
-
-    public static void main(String[] args) throws IOException {
-        Consola consola = new Consola();
-        Cliente cliente = new Cliente();
-        CC cc = new CC ();
-        CAPesos ca = new CAPesos();
-        CADolares dol = new CADolares();
-        Movimiento movimiento = new Movimiento();
-        LectorDeArchivos lector = new LectorDeArchivos();
-        
-        //Bienvenida y menu principal
-        String numTarjeta = consola.bienvenida();
-        String pin =  consola.pin();
-        cliente.validacion(numTarjeta, pin);
-        for (Cuenta cuenta: cliente.getMisCuentas()){
-            if(cuenta.getTipo() == 1){
```

Comment:
Qué es tipo 1? esto deberia ser un enum
https://www.w3schools.com/java/java_enums.asp#:~:text=An%20enum%20is%20a%20special,the%20constants%20with%20a%20comma.